### ISSUE TITLE
AL and Machine learning - Question on scope

### ISSUE THREAD

**Issue by joshueoconnor** [w3c/ai-accessibility#3]
I don't think the word ‘clearly' should be used here. This document could be better as a question, 'What is the current bearing of AI on accessibility?' or a gap analysis.

Remove ‘clearly'

"This is a draft collection of relevant information related to cross-disability accessibility guidance of how developments in machine learning and generative Artificial Intelligence (AI) clearly bears an impact on web accessibility standards and processes. Given the rapid changes in the consumption and development of AI design, this is intended to be a starting point to group the accessibility implications of machine learning and generative AI technologies.”

**Comment by sehollier** [comment]
Agreed.
